code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -102,10 +102,10 @@
 /obj/item/weapon/storage/backpack/ww2/jap/ammo_crate
 	name = "japanese ammo crate"
 	desc = "It's a crate equipped with straps for carrying, often used by assistant gunners."
-	icon_state = "japammo"
-	item_state = "japammo"
-	worn_state = "japammo"
-	base_icon = "japammo"
+	icon_state = "ammo_crate"
```

Comment:
Was japammo inexistant?

Reply:
Nope it's exists it's just another thing he's arbitrarily changing the name of.

Reply:
*existed, he changed it in the .dmi